dev-util/perf/perf-5.10.16.ebuild

```diff
@@ -198,6 +198,7 @@ perf_make() {
 		NO_ZLIB= \
 		WERROR=0 \
 		LIBDIR="/usr/libexec/perf-core" \
+		libdir="/usr/$(get_libdir)" \
```

Comment:
Shouldn't this be `lib=$(get_libdir)` as mentioned in the bug? I need to check if this works on prefix or not before merging.

Reply:
`lib=$(get_libdir)` will install those so files to `imagelib/traceevent/plugins/`.

Reply:
Really? I tried lib=lib and I get the expected:
`/usr/lib/traceevent/plugins/plugin_tlb.so`
On prefix with lib=test:
`/home/ionen/prefix/usr/test/traceevent/plugins/plugin_tlb.so`
Not that I tested all that much.

Reply:
We do want those in `usr/$(get_libdir)/traceevent/plugins`, right? These are not general libraries, but plugins only used by perf, so they are not meant to go straight into `/usr/lib64` or `/usr/lib` like `libperf-*.so`. The fix is to use `lib` rather than `lib64` with musl as far as I can tell from the bug. Howver, I guess your patch is equivalent as it doesn't change `plugindir` in any case.